cherab/openadas/openadas.py

```diff
@@ -86,6 +86,21 @@ def recombination_rate(self, ion, charge):
 
         return RecombinationRate(data, extrapolate=self._permit_extrapolation)
 
+    def thermalchargeexchange_rate(self, ion, charge):
+
+        if isinstance(ion, Isotope):
+            ion = ion.element
+
+        try:
+            data = repository.get_recombination_rate(ion, charge)
```

Comment:
Looks like a typo, shouldn't this be `data = repository.get_thermalchargeexchange_rate()`. You should be getting weird results with it like this.